pkg/util/utils.go

```diff
@@ -19,11 +20,6 @@ func Success(format string, a ...interface{}) {
 	color.Cyan(format, a...)
 }
 
-// Warning will present the user with warning text.
-func Warning(format string, a ...interface{}) {
```

Comment:
I wondered that myself. I ended up removing it because:

- We're not using it right now
- It's trivial code to add if we need it in the future

I'm happy either way.